sbt-plugin/src/main/scala/akka/grpc/sbt/AkkaGrpcPlugin.scala

```diff
@@ -93,11 +93,15 @@ object AkkaGrpcPlugin extends AutoPlugin {
         // hack to get our (dirty) hands on a proper sbt logger before running the generators
         generatorLogger.logger = streams.value.log
         (PB.recompile in Test).value
-      })
+      },
+      target in akkaGrpcCodeGeneratorSettings := crossTarget.value / "akka-grpc")
```

Comment:
Isn't one issue with this that IntelliJ won't pick it up automagically anymore?

Reply:
IntelliJ adds all `managedSourceDirectories` as generated source directories, so since we add this to that, it gets picked up.

Reply:
Actually, maybe you're right. We should get that fixed in IntelliJ though.... needing to manually mark the directory as a generated sources root in intellij is better than a broken build... and the Scala compiler is really weird when you tell it to compile a non existent file, here's the error message you get:

```
error while decoding /home/travis/build/cloudstateio/cloudstate/java-support/target/src_managed/main/sbt-buildinfo/BuildInfo.scala with UTF-8: /home/travis/build/cloudstateio/cloudstate/java-support/target/src_managed/main/sbt-buildinfo/BuildInfo.scala (No such file or directory)
Please try specifying another one using the -encoding option
one error found
```